compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/SkiaBasedOwner.skiko.kt

```diff
@@ -136,11 +133,10 @@ internal class SkiaBasedOwner(
     override val root = LayoutNode().also {
         it.layoutDirection = layoutDirection
         it.measurePolicy = RootMeasurePolicy
-        it.modifier = semanticsModifier
+        it.modifier = EmptySemanticsElement
             .then(focusOwner.modifier)
             .then(keyInputModifier)
-            .onPreviewKeyEvent(onPreviewKeyEvent)
-            .onKeyEvent(onKeyEvent)
+            .then(modifier)
```

Comment:
> not sure what `EmptySemanticsElement` do

Well, during one of iterations I removed it (because we already passing semantics for `popup` and `dialog`), but it led to failing 375 tests 🙈 with
```
java.lang.NullPointerException
	at androidx.compose.ui.semantics.SemanticsNodeKt.SemanticsNode(SemanticsNode.kt:534)
	at androidx.compose.ui.semantics.SemanticsOwner.getRootSemanticsNode(SemanticsOwner.kt:36)
```